clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -137,6 +136,17 @@ abstract class OpenstackClientProvider {
 
 
   /**
+   * Deletes a security group.
+   *
+   * @param securityGroupId id of the security group
+   */
+  void deleteSecurityGroup(String securityGroupId) {
+    handleRequest(AtomicOperations.DELETE_SECURITY_GROUP) {
```

Comment:
We should be using the new exception handling strategy that @drmaas introduced in his latest PR ... Throwing OpenstackProviderException instead of the handleRequest by operation.